Remove legacy content packs (#562)
Graylog2/documentation: pages/upgrade/graylog-3.0.rst

Furthermore, it's not possible anymore to enable both settings (SMTP with STARTTLS and SMTP over SSL) at the same time because this led to errors at runtime when Graylog tried to upgrade the connection to TLS with STARTTLS in an already existing SMTPS connection.

Most SMTP services prefer SMTP with STARTTLS to provide an encrypted connection.

Legacy Content Packs
====================

The implementation of content packs where fundamentally reworked. Parameters

> were add and checks implemented to give the user a better usability.

Reviewer: A couple of suggestions, but take into account I'm no native speaker 😉 
```suggestion
were added and checks implemented to give the user better usability.
```

This rework did come with the cost that old content packs might not
work any longer and stop the new content packs from loading.
If the content packs page does not finish loading we recommend to remove

> the old content packs from your mongodb.

Reviewer: ```suggestion
the old content packs from your MongoDB.
```

> For that, please connect to your mongodb and remove all content packs

Reviewer: ```suggestion
For that, please connect to your MongoDB shell and remove all content packs
```

> with following command:

Reviewer: ```suggestion
with the following command:
```
